CMakeLists.txt

```diff
@@ -389,44 +389,45 @@ list(APPEND JIT_TEST_SRCS
   ${NVFUSER_SRCS_DIR}/kernel_db/test/test_nvfuser_kernel_db_open.cpp
   ${NVFUSER_SRCS_DIR}/kernel_db/test/test_nvfuser_kernel_db_query.cpp
   ${NVFUSER_SRCS_DIR}/kernel_db/test/test_nvfuser_kernel_db_write.cpp
+  ${NVFUSER_ROOT}/test/test_alias.cpp
```

Comment:
I sorted the files -- no behavior change.

Reply:
Just curious: do you do this manually, or there is some tool for doing so?

Reply:
In vim, select the lines and :sort